File: num_g.c
#include<stdio.h>
#include<math.h>
#define pi 3.1415926535
int main()
{
  float n[1000];
  n[0]=0.2;
  float alpha = pi;
  int k=1;
  float num;
  while(k<1000){
    num = (n[k-1]+alpha)*100;
    int num_temp = num;
    n[k] = num-num_temp;
    k++;
  }
  int i;
  for(i=0;i<1000;i++){
    printf("%d - %.4f \n",i,n[i]);
  }
}

File: num_g.py
import math
n=list()
n.append(0.2)
a=math.pi
for k in range(1,1000):
    num = (n[k-1]+a)*100
    n.append(num-int(num))

for i in range(0,1000):
    n[i] = "{:.4f}".format(n[i])

print(n)
print(len(n))
